var mod = angular.module("externalDsrpMod", []);

mod.controller("externalDsrpCtrl", ["$scope", function($scope) {

    var DIR = 'images/dsrp/'

    $scope.labs = [
        {
            name: 'Ziv Bar-Joseph – Department of Computer Science, Carnegie Mellon University',
            projects: [
                {
                    title: 'Target prediction using LINCS data ',
                    text: 'A major challenge in drug discovery is to identify a potential inhibitor for protein targets that is both specific and effective. In collaboration with the Camacho Lab at the University of Pittsburgh we are using LINCS data to predict such targets for the tens of thousands of small molecules that have profiled in the datasets. For this, we are using a training set of known targets for a small set of drugs and developing classifiers that, based on the expression response following treatment with the drug predict potential targets based on the similarity of their expression profiles to the drug expression response. To further improve the accuracy we are utilizing a large cohort of protein interaction data and using information from modules to refine the rankings of potential targets. The genomics and interaction data is then used to rank potential targets and the top 100 targets for each small molecule are than further analyzed using docking to determine which are likely to bind the molecule / drug. For proteins of interest we would identify the top predicted molecules and would be experimentally test the impact of these molecules on the protein.'
                },
                {
                    title: 'Multi-task learning of cancer pathways',
                    text: 'A key goal of the methods we develop is to reconstruct the signaling and regulatory pathways that are activated in cancer cells prior and after treatment. While each cell type may respond differently to a specific drug, it is likely that these responses significantly overlap since the pathways targets by the drug are usually the same in all cells. To accurately model these pathways we are developing methods that integrate information from multiple types of cells to reconstruct such core response pathways while at the same time also allowing for a cell specific component. Such a process is termed multi-task learning in the machine learning terminology and while it was extensively applied in the part to classification problems it has rarely been applied to model and reconstruct response networks. In our case we are integrating the LINCS data with additional interaction data including protein-protein and protein-DNA and further using the data to infer new protein-DNA interactions from the knockdown experiments. Preliminary results indicate that multi-task learning improves upon both – models constructed from a single cell type data and models constructed by using the union of all cell types (without allowing for cell specific component).'
                }
            ]
        },
        {
            name: 'Peter Hornbeck – Cell Signaling Technology Inc.',
            projects: [
                {
                    title: 'Curation and Computational Access to Cancer TMT Datasets via PhosphoSitePlus',
                    text: 'We have accomplished work that will enable PhosphoSitePlus to serve the proteomic cancer datasets including: 1) set up a server in the PhosphoSItePlus (PSP) cluster for running CORE, the analytical suite of software for analyzing MS2 raw data. CORE includes routines that apply advanced and regularly updated algorithms for peptide and site assignments developed in the Gygi lab at Harvard University. 2) Implemented new processes for displaying MS spectra derived from experiments performed at CST. This will enable us to serve not only individual spectra for individual sites from within the UI, but also to include spectra in bulk processing via the API to be developed in the coming year. Implementation of this feature involved developing a Java parser of the spectral data, and implementation of ‘lorikeet’, a relatively recent versatile JQuery plugin to view MS/MS spectra annotated with fragment ions. 3) Developed processes that support importing, storing, and serving TMT data in PSP. This involved analysis of the CORE data structures, modifying the PSP Oracle DB structure, and programming the transfer and storage of TMT data. We are in the process of beta testing this process with the perturbagen lung cancer cell line data. 4) Struts and JSP migration. The architecture of the application was designed in 2006 and uses Struts 1. In order to support future growth, and to accommodate interactions via the API, it is essential to migrate to Struts 2.  80% of this migration has been completed. All Struts 1 action classes have been migrated to Struts 2 classes. The migration and implementation will require five more weeks: two weeks to complete the JSP migration and testing, two weeks to merge with the existing code base, and one week to complete testing. 5) Curation of MS2 experiments. Supplementary datasets from approximately 12 cancer or perturbagen MS2 publications, encompassing over 10,000 peptides and 2,000 new sites were curated into PSP. Additionally, we have begun the process of curating the CST cancer cell line & perturbagen TMT data into PSP. Perturbagens include Gleevec, Iressa, crizotinib, and geldanomycin.'
                },
                {
                    title: 'Network Analysis of Multi-layered PMTs Collected from 31 NSCLC Cell-Lines',
                    text: 'Using enrichment analysis we obtained kinase signatures for each lung cancer cell line.  We selected the top up- and down-regulated phosphosites for each cell line, defined as the top 10% of up- or down-regulated phosphosites, and performed enrichment analysis for kinases using our kinase-substrate library at the phospho-site level. Significantly enriched kinases, with a Benjamini-Hochberg corrected p-value of less than 0.05, were discovered for each cell line. We summarized our enrichment results in an interactive web-based kinase enrichment connectivity map. The enrichment map has kinases as rows and cell lines as columns. Each tile represents an enriched kinase in a NSCLC cell line and the opacity of the tile is proportional to the Benjamini-Hochberg corrected p-value. Some kinases are enriched based on up- and down-regulated phosphosites in a single cell line and in this case the tile is split diagonally, with the red (up) triangle on the top and the blue (down) triangle on the bottom. Our enrichment results reveal kinase signatures of NSCLC cell lines and point to potential new driver kinases in NSCLC cell lines. Our results also agree with previous knowledge about kinase mutations/fusion/copy-number-variation. For instance, three cell lines that are known to have activating EGFR mutations: H1650, HCC827, and H1975, all show enrichment for up-regulated phosphosites that are substrates of EGFR. Furthermore, these three cell lines have the highest enrichment for EGFR of all 31 cell lines. The cell lines H3122 and H2228 have activating fusions of ALK, and these cell lines are the only two that are enriched for ALK. Finally, the cell line H1703, which has aPDGFRA amplification, is enriched for PDGFRA and the cell line HCC78, which has an activating ROS1 fusion, is exclusively enriched for ROS1. These results indicate that our phosphoproteomics data and enrichment method are able to accurately recover known biological information on these cell lines and that we can be confident of the predictions made by our results.',
                    image: DIR + 'fig3.png',
                    caption: 'Fig. 3 expression map visualizes NSCLC cell lines and z-score normalized expression of kinases from the CCLE. A red tile indicates up-regulated expression of this gene relative to the other NSCLC cell lines and blue indicates down-regulated. The cell lines and kinases have been clustered.'
                }
            ]
        },
        {
            name: 'Ka Yee Yeung – Institute of Technology, University of Washington',
            projects: [
                {
                    title: 'Exploration of LINCS data',
                    text: 'We have extended our in-house utilities by developing a browser-based interface to extract slices of the L1000 data to facilitate downstream analyses. This interface is written in php, runs on all operating systems and mobile devices. The goal is to enhance the data mining effort among Computer Science students without background in biology or bioinformatics. We also have started a project to compute prior probabilities of regulatory relationships using external data sources (in addition to LINCS data) so as to guide the search of candidate regulators in our regression-based network inference methods.'
                },
                {
                    title: 'Artifacts of L1000 data',
                    text: 'We study the effect of various experimental design variables on the statistical properties of the L1000 level 3 data. We observed that different bead batches often exhibit different data distributions. Since each bead measures the expression levels of two genes in the L1000 Luminex platform, we observed that the L1000 deconvolution procedure occasionally results in the expression levels of the two genes to be switched and assigned to the wrong gene. We are currently investigating a model-based clustering method utilizing flipping of clusters to detect and correct these errors. This will hopefully improve the quality of benchmarks and subsequent analyses.'
                },
                {
                    title: 'Improve scalability of network inference algorithms',
                    text: 'We have previously developed probabilistic regression-based algorithms [Yeung et al. PNAS 2011, Lo et al. BMC Systems Biology 2012, Young et al. BMC Systems Biology 2014] to infer gene networks from gene expression data. In order to apply our network inference methods to LINCS data consisting of millions of experiments, we have improved the scalability of our current implementation. The major improvement comes from the use of more scalable and efficient matrix decomposition algorithms. Combined with the use of faster FORTRAN linear algebra libraries, conversion of R routines to C++, more efficient data structures, optimized hash functions, we achieve a 100 fold improvement in speed. The algorithm has also been parallelized to use multiple cores and processors for additional acceleration and scalability. In another project, we are developing a graph theory algorithm to remove redundant edges after our regression-based network is computed.'
                }
            ]
        }
    ];
}]);
